CHANGELOG.rst

```diff
@@ -1,10 +1,11 @@
 Changelog
 =========
 
-6.4.0 (unreleased)
+6.4.0 (2023-03-21)
 ------------------
 
-- Nothing changed yet.
+- Upgrade pyramid-storage, which allows for Google Cloud Storage (GCS) integration (`#615 <https://github.com/Kinto/kinto-attachment/pull/615>`_)
```

Comment:
I wouldn't mention it in the changelog since we didn't change anything to the lib.